dali/operators/python_function/dltensor_function.cc

```diff
@@ -28,10 +28,13 @@ DALI_SCHEMA(DLTensorPythonFunctionImpl)
     .NumInput(0, 256)
     .OutputFn([](const OpSpec &spec) {return spec.GetArgument<int>("num_outputs");})
     .NoPrune()
+    .Unserializable()
```

Comment:
Please add this to the ExternalSource as well.

Reply:
done